### ISSUE TITLE
Sentence textWrap only if columns>0

### ISSUE THREAD

**Issue by cmhughes** [cmhughes/latexindent.pl#188]
During testing of https://github.com/cmhughes/latexindent.pl/issues/185 I came across the following:

mwe.tex
-
```
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec consectetur ac nulla eu ultrices integer ac mi et ligula pulvinar rutrum nam tristique sed tortor eget iaculis vamus aliquam venenatis neque. In hac habitasse platea dictumst. Pellentesque sit amet magna non tortor porta rhoncus eget non diam interdum et malesuada fames ac ante ipsum primis in faucibus. Cras sit amet placerat lacus ut porttitor eleifend velit ac laoreet nullam ac finibus erat, a auctor metus. 
```
with the YAML

cmh.yaml
-
```
modifyLineBreaks:
    oneSentencePerLine:
        manipulateSentences: 1 
        removeSentenceLineBreaks: 1
        textWrapSentences: 1
```
and running
```
latexindent.pl -m mwe.tex -l=cmh.yaml
```
gives the same error repeated:
```
Unescaped left brace in regex is deprecated here (and will be fatal in Perl 5.30), passed through in regex; marked by <-- HERE in m/\G((?:(?=[^\n])\X){ <-- HERE 0,-1})((?=\s)\X|\n+|\z)/ at /home/cmhughes/perl5/perlbrew/perls/perl-5.28.1/lib/5.28.1/Text/Wrap.pm line 62.
Unescaped left brace in regex is deprecated here (and will be fatal in Perl 5.30), passed through in regex; marked by <-- HERE in m/\G((?:(?=[^\n])\X){ <-- HERE -1})/ at /home/cmhughes/perl5/perlbrew/perls/perl-5.28.1/lib/5.28.1/Text/Wrap.pm line 67.
```

The fix is to ensure that `columns>0`, so, for example:
```
modifyLineBreaks:
    textWrapOptions:
         columns: 80
    oneSentencePerLine:
        manipulateSentences: 1 
        removeSentenceLineBreaks: 1
        textWrapSentences: 1
```

However, this shouldn't be necessary, and the script should know to skip text wrapping if `columns` is not greater than 0. I think that the specific part that needs attention is in `Sentence.pm` and the line `if(${$masterSettings{modifyLineBreaks}{oneSentencePerLine}}{textWrapSentences}){` which needs an additional condition to ensure that `columns>0`, or else otherwise to set a default value for `columns` to (I think) 80.

**Comment by cmhughes** [comment]
Resolved as of https://github.com/cmhughes/latexindent.pl/commit/bb2ad163aff49b7e67d7a516807a881d578745fd. From the updated documentation:

*If you specify \texttt{textWrapSentences} as 1, but do \emph{not} specify a value for \texttt{columns}
then the text wrapping will \emph{not} operate on sentences, and you will see a warning in \texttt{indent.log}.*

**Comment by cmhughes** [comment]
Implemented in `Version 3.8`, https://github.com/cmhughes/latexindent.pl/pull/190
